Fix:

error: implicit declaration of function 'math_init' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://github.com/davidt/Fyre/pull/2
--- src/main.c.orig	2006-06-22 01:02:02.000000000 -0500
+++ src/main.c	2024-09-11 20:47:18.000000000 -0500
@@ -47,6 +47,7 @@
 #include "remote-server.h"
 #include "batch-image-render.h"
 #include "gui-util.h"
+#include "math-util.h"
 
 #ifdef HAVE_GNET
 #include "cluster-model.h"
